sdk/tracing/azure-core-tracing-opencensus/src/samples/java/com/azure/core/tracing/opencensus/HelloWorldAsync.java

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.tracing.opencensus;
+
+import com.azure.storage.queue.QueueAsyncClient;
+import com.azure.storage.queue.QueueClientBuilder;
+import io.opencensus.common.Scope;
+import io.opencensus.exporter.trace.zipkin.ZipkinTraceExporter;
+import io.opencensus.trace.Tracer;
+import io.opencensus.trace.Tracing;
+import io.opencensus.trace.config.TraceConfig;
+import io.opencensus.trace.config.TraceParams;
+import io.opencensus.trace.samplers.Samplers;
+import reactor.util.context.Context;
+
+import static com.azure.core.util.tracing.Tracer.OPENCENSUS_SPAN_KEY;
+
+/*
+ *  This example shows tracing support in azure-storage-queue sdk using azure-core-tracing plugin package.
+ */
+public class HelloWorldAsync {
+    private static final String ACCOUNT_NAME = System.getenv("AZURE_STORAGE_ACCOUNT_NAME");
+    private static final String SAS_TOKEN = System.getenv("PRIMARY_SAS_TOKEN");
```

Comment:
I believe service specific environment variables should be named with AZURE prefix.
https://azure.github.io/azure-sdk/general_implementation.html#service-specific-environment-variables

Reply:
updated